snooty/parser.py

```diff
@@ -1177,6 +1178,41 @@ def __init__(
 
         self.config.substitution_nodes = substitution_nodes
 
+        # Parse banner value and instantiate a banner node for postprocessing, if a banner value is defined.
```

Comment:
What usage would we expect in snooty.toml? I'm picking up something like this:

```toml
banner = { target = "???", variant = "info" , value = "Deprecated page" }
```

What does `target` represent, mainly, and what does the usage look like?

Reply:
`target` would be a glob pattern for file selection - though admittedly this is mostly for implementation convenience.

But as an interface, I do think there's value in using glob pattern - ideally, a writer would be able to specify something like

```
[banner]
target = "*"
variant = "warning"
value = "This product is deprecated"
```
for whole repo application of a banner, or something like `target = '*/get-started*` for application across all pages rendered from the `get-started` directory.

I have logic in the postprocessor to exclude `includes` from being targeted as well.

Reply:
I think all of this makes sense (including the glob thing) with one suggestion to maybe change the labels so that the writers will have an easy time making these updates, what about something like:

```
[banner]
files = "*"
variant = "warning"
content = "This product is deprecated"
```

In terms of the target/pages field -- that's relative to `source/` right?

Reply:
What if we need a banner to be applied to a set of disjoint globs? What if we need multiple banners, does this permit that?

Reply:
In my latest changeset I've added support for specifying multiple banners and for specifying multiple glob targets for each banner - it was a change I'd thought about adding while implementing, and turned out to be simple enough to just add in 😄 .

I'm currently adding logic to ensure that we match targets relative to `source/`, then this should be good for another pass.